src/components/modals/unjail-unstake/unjailUnstake.jsx

```diff
@@ -52,8 +55,8 @@ export default function UnjailUnstake({
       if (txResponse.txhash !== undefined) {
         setPassphrase("");
         const publicKeyHex = unlockedAccount.privateKey.toString("hex");
-        pocketService.saveUserInCache(account.addressHex, publicKeyHex, ppk);
-        pocketService.saveTxInCache(
+        updateUser(account.addressHex, publicKeyHex, ppk.toString());
```

Comment:
What is ppk that we need to cast it to a string? If it's a buffer, it needs to be `.toString('hex')`—this might have caused some issues.

Reply:
ppk is just a string, maybe it's casted for safety to defend against some edge case that I'm not aware of.

Reply:
Ah right, but it's the PPK, so it's why it's not casted to hex, I understand now.